Makefile

```diff
@@ -522,15 +520,35 @@ scan-artifacts: telekube
 #
 .PHONY: telekube
 telekube: GRAVITY=$(GRAVITY_OUT) --state-dir=$(PACKAGES_DIR)
-telekube: $(GRAVITY_BUILDDIR)/telekube.tar
+telekube: $(TELEKUBE_OUT)
 
-$(GRAVITY_BUILDDIR)/telekube.tar: packages
+$(TELEKUBE_OUT): packages
 	GRAVITY_K8S_VERSION=$(K8S_VER) $(GRAVITY_BUILDDIR)/tele build \
 		$(ASSETSDIR)/telekube/resources/app.yaml -f \
 		--version=$(TELEKUBE_APP_TAG) \
 		--state-dir=$(PACKAGES_DIR) \
 		--skip-version-check \
-		-o $(GRAVITY_BUILDDIR)/telekube.tar
+		-o $(TELEKUBE_OUT)
+
+$(GRAVITY_OUT):
+	$(MAKE) -C build.assets build
```

Comment:
Nit: This will build all binaries, not just `gravity`.

Reply:
What is the correct way to build just `GRAVITY_OUT`?  I am having a deal of trouble figuring that out, because the binary targets are lumped together under a parallel dockerized make in `build-in-container` -- which I didn't really want to take as a pretty slow (2m22s on my system) `.PHONY` dependency.

This `$(GRAVITY_OUT)` target allows me to cheat and make sure a binary exists on the FS without taking the costly `.PHONY` dependency.  It is still prone to failure because it in turn doesn't depend on any of the upstreams that feed into `$(GRAVITY_OUT), and thus won't get rebuilt if a source changes -- but that is rampant in our makefiles and not a problem I want to tackle in this PR.

Reply:
Hmm, IIRC there's a `BINARIES` variable that sets the binaries to build so you could try to use something like

```
BINARIES=gravity $(MAKE) -C build.assets build
```

Reply:
Fixed in beeb7c5.  This may lead to non-parallel rebuilds in some cases.